design.md

```diff
@@ -2,12 +2,25 @@
 
 ## Who is the target for this design, e.g., are you assuming any knowledge on the part of the language users?
 
+I'm assuming the user will either be familiar with Picobot or won't (so, everyone lol), but it is designed with natural language in mind to be easily understood and accessible to users.
+
 ## Why did you choose this design, i.e., why did you think it would be a good idea for users to express the maze-searching computation using this syntax?
 
+I choose this design to give flexibility to users of the DSL. Despite being nearly done with the Mudd CS major, I completely forgot Picobot and had to relearn it. Based on this, I changed the language to be both easily understood as well as give the user flexibility if they want to use syntax from previous Picobot and this new syntax.
```

Comment:
I had a similar experience. Picobot was my favorite aspect of CS5 but I had completely forgotten about it and how it worked. Relearning it and thinking about ways that the syntax could be improved was something that I enjoyed. Since its not too hard to pick back up, I liked how you allowed for users to use elements of the original Picobot syntax along with a different syntax for others